code/modules/jobs/job_types/roguetown/Inquisition/puritan.dm

```diff
@@ -25,9 +26,163 @@
+
+	category_tags = list(CTAG_PURITAN)
+
+/datum/outfit/job/roguetown/puritan/inspector/pre_equip(mob/living/carbon/human/H)
 	..()
 	shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/puritan
+	armor = /obj/item/clothing/suit/roguetown/armor/leather/heavy/coat
+	belt = /obj/item/storage/belt/rogue/leather/knifebelt/black/psydon
+	neck = /obj/item/clothing/neck/roguetown/psicross/silver
+	shoes = /obj/item/clothing/shoes/roguetown/boots
+	wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+	pants = /obj/item/clothing/under/roguetown/tights/black
+	cloak = /obj/item/clothing/cloak/cape/puritan
+	backr = /obj/item/storage/backpack/rogue/satchel/black
+	beltr = /obj/item/storage/belt/rogue/pouch/coins/rich
+	head = /obj/item/clothing/head/roguetown/puritan
+	gloves = /obj/item/clothing/gloves/roguetown/leather
+	beltl = /obj/item/rogueweapon/sword/rapier
+	backpack_contents = list(/obj/item/storage/keyring/puritan = 1, /obj/item/lockpickring/mundane = 1, /obj/item/rogueweapon/huntingknife/idagger/silver)
+	if(H.mind)
+		H.mind.adjust_skillrank(/datum/skill/combat/swords, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/knives, 4, TRUE) //Keeping it up with Confessor
+		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 4, TRUE) //Keeping it up with Confessor
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 4, TRUE) //Keeping it up with Confessor
+		H.mind.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/sewing, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 3, TRUE) //Keeping it up with Confessor
+		H.mind.adjust_skillrank(/datum/skill/misc/lockpicking, 5, TRUE) ///Ortho Confessor gets master in every stealth skill. Insane. 
+		H.mind.adjust_skillrank(/datum/skill/misc/tracking, 5, TRUE) //To track down heretics to their evil lairs... 
+		H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 5, TRUE) //Always thought it was crazy they got listening devices but NO means to effectively sneak them into places.
+		if(H.age == AGE_MIDDLEAGED)
+			H.mind.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
+			H.change_stat("perception", 1)
+		if(H.age == AGE_OLD)
+			H.mind.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 1, TRUE)
+			H.change_stat("perception", 2)
+			H.change_stat("strength", 2) //to offset the age
+		H.change_stat("strength", 1) ///They deserve the whips too actually
+		H.change_stat("endurance", 3) ///Who the fuck gave Confessor 3 Endurance lmao
+		H.change_stat("perception", 3)
+		H.change_stat("speed", 3) //Rogues get 3 fucking speed and spymaster hands get 4, they deserve this. Confessor does too wtf
```

Comment:
rogues also only get +6/-1 statswise, while you have given this class 10 with no negatives BEFORE age adds another one or four to their stats. This is insane. by comparison guard captain gets +10 without age buffs and RG gets +9/-1 before locational buffs. also generally speaking no class gets to ignore old age for free, they instead get other stat or skill bonuses.

Reply:
Yeah I dont care enough to argue I'll just change it so it gets merged